lib/trino-filesystem/src/main/java/io/trino/filesystem/tracing/Tracing.java

```diff
@@ -21,17 +21,17 @@
 
 import java.util.Optional;
 
-final class Tracing
+public final class Tracing
 {
     private Tracing() {}
 
-    public static <T> Attributes attribute(AttributeKey<T> key, Optional<T> optionalValue)
```

Comment:
If we're going to make this class public, why not keep all of the methods as public? It feels weird to hide some of them, just because they aren't used externally now.

Reply:
made it simply public now